source/content/terminus/updates.md

```diff
@@ -40,14 +51,18 @@ releases.
 
 ### Nothing to install or update
 
-If the update command above returns output indicating that no updates were found, delete the existing Terminus version (e.g. <code>\$HOME/terminus</code>) and re-run the install command:
+For Composer-managed Terminus installations, if the update command above returns output indicating that no updates were found, delete the existing Terminus version (e.g. <code>\$HOME/terminus</code>) and re-run the install command:
 
 ```bash{promptUser: user}
 rm -rf $HOME/terminus mkdir $HOME/terminus cd $HOME/terminus curl -O
 https://raw.githubusercontent.com/pantheon-systems/terminus-installer/master/builds/installer.phar
 php installer.phar install
 ```
 
+### Self:update not defined
+
+The `self:update` command is only available for standalone Terminus installed using the [standalone Terminus PHAR](/terminus/install#standalone-terminus-phar). If `self:update` returns a not defined error, use the [Terminus Installer PHAR](#update-terminus-installer-phar) update instructions above.
```

Comment:
@carolyn-shannon I saw that you added the partial file with this info - is it meant to take this paragraph's place?

Reply:
@EdwardAngert that partial file is under the `terminus` subdir and matches a terminus command, which means it's built into the command reference page for `terminus self:update`.